operations/image_management/Configure_IMS_to_validate_rpms.md

```diff
@@ -88,16 +211,85 @@ Configuring the Image Management Service (IMS) to validate the GPG signatures of
     Dockerfile  entrypoint.sh  my-signing-key.asc
     ```
 
+1. (`ncn-mw#`) (CSM 1.5.0 and later) Verify that the following file is in the temporary directory.
+
+    ```text
+    armentry.sh
+    ```
+
+1. (`ncn-mw#`) (CSM 1.5.2 and later) Verify that the following file is in the temporary directory.
+
+    ```text
+    remote_build_entrypoint.sh
+    ```
+
+1. (`ncn-mw#`) (CSM 1.5.0 and later with aarch64 hardware) Install QEMU emulation software.
+
+    For cross compiling aarch64 images, QEMU emulation software must be installed on the
+    node where this operation is taking place. If QEMU is already installed this step may
+    be skipped.
+
+    1. Download and install the QEMU emulation package.
+
+        ```bash
+        wget https://github.com/multiarch/qemu-user-static/releases/download/v7.2.0-1/qemu-aarch64-static
+        mv ./qemu-aarch64-static /usr/bin/qemu-aarch64-static
+        chmod +x /usr/bin/qemu-aarch64-static
+        ```
+
+    1. Set up `binfmt_misc` for handling emulation.
+
+        ```bash
+        if [ ! -d /proc/sys/fs/binfmt_misc ] ; then
+            echo "- binfmt_misc does not appear to be loaded or isn't built in."
+            echo "  Trying to load it..."
+            if ! modprobe binfmt_misc ; then
+                echo "FATAL: Unable to load binfmt_misc"
+                exit 1;
+            fi
+        fi
+        ```
+
+    1. Mount the emulation file system.
+
+        ```bash
+        if [ ! -f /proc/sys/fs/binfmt_misc/register ] ; then
+            echo "- The binfmt_misc filesystem does not appear to be mounted."
+            echo "  Trying to mount it..."
+            if ! mount binfmt_misc -t binfmt_misc /proc/sys/fs/binfmt_misc ; then
+                echo "FATAL:  Unable to mount binfmt_misc filesystem."
+                exit 1
+            fi
+        fi
+        ```
+
+    1. Register QEMU for aarch64 emulation.
+
+        ```bash
+        if [ ! -f /proc/sys/fs/binfmt_misc/qemu-aarch64 ] ; then
+            echo "- Setting up QEMU for ARM64"
+            echo ":qemu-aarch64:M::\x7f\x45\x4c\x46\x02\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\xb7\x00:\xff\xff\xff\xff\xff\xff\xff\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/bin/qemu-aarch64-static:F" >> /proc/sys/fs/binfmt_misc/register
+        fi
+        ```
+
 1. (`ncn-mw#`) Using the `podman` command, build and tag a new `cray-ims-kiwi-ng-opensuse-x86_64-builder` image.
 
+    For versions prior to CSM 1.5.0 or for systems with only `x86_64` hardware, use the following command:
+
+    ```bash
+    podman build -t registry.local/artifactory.algol60.net/csm-docker/stable/cray-ims-kiwi-ng-opensuse-x86_64-builder:${KIWI_VERSION}-validate .
+    ```
+
+    For CSM 1.5.0 or later and systems that include aarch64 hardware, use the following command:
```

Comment:
Since this is going into the CSM 1.6 docs, isn't the "CSM 1.5.0 and later" part unnecessary?
```suggestion
    For systems that include aarch64 hardware, use the following command:
```